src/tests/end-to-end/test-resources/clean.html

```diff
@@ -0,0 +1,15 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
```

Comment:
are we using this only for manual validation or will this be used in e2e flow as well?

Reply:
As of this PR, only for manual validation